src/kazagumoSearch.js

```diff
@@ -32,7 +32,8 @@ class kazagumoSearch {
         const sources = {
             "youtube": "yt",
             "youtube_music": "ytm",
-            "soundcloud": "sc"
+            "soundcloud": "sc",
```

Comment:
Codacy has a fix for the issue: [Replace `············"soundcloud"` with `······soundcloud`](https://app.codacy.com/gh/Takiyo0/Kazagumo/pullRequest?prid=9683949)
```suggestion
      soundcloud: "sc",
```